src/simple_server/core.clj

```diff
@@ -2,38 +2,72 @@
   (:require [clojure.pprint]
             [ring.adapter.jetty :refer [run-jetty]]
 
-            [ring.util.response :refer [response created redirect not-found status]]
-            [compojure.core :refer [GET POST ANY defroutes]]
+            [ring.util.response :refer [response created redirect not-found status content-type]]
+            [compojure.core :refer [GET POST PUT ANY defroutes]]
             [compojure.coercions :refer [as-int]]
             [ring.middleware.defaults :as middleware]
+            [ring.middleware.cookies :refer [wrap-cookies]]
 
             [ring.mock.request :as mock]
-            [simple-server.simple-game :as game]))
+            [simple-server.simple-game :as game]
+            [clojure.edn :as edn]))
 
 ;;; Finally, let us truly separate concerns between our "application code"
 ;;; and our "http code".  Our game now lives in its own namespace, and
 ;;; is fully testable independent of our "presentation layer".
 
-(defn new-game-handler []
-  (when (game/new-game!)
-    (response "OK- start guessing at /guess")))
+(def users-info
+  "Store all users' login information"
+  (atom {}))
```

Comment:
Maybe just a set of usernames?

Reply:
Yup

Reply:
Kind like it. But I was trying to do it like... For example if yunfeng login to this application then the users-info will be like {"yunfeng": true}. But I think it's the same thing.